conf/frpc_full.ini

```diff
@@ -68,6 +68,9 @@ tls_enable = true
 meta_var1 = 123
 meta_var2 = 234
 
+# specify udp packet size, unit is byte
+udp_packet_size = 1024
```

Comment:
udp_packet_size = 1500

Reply:
More comments. 

This parameter should be same between client and server.

It affects the udp and sudp proxy.